src/App.tsx

```diff
@@ -15,47 +16,112 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+// Use this function in the render to prepare goods
+export function getReorderedGoods(
```

Comment:
```suggestion
export function getPreparedGoods(
```
If you name your function right, then there will be no need for a comment. 
Try to make your code clear and understandable to other developers without any comments.